providers.tf

```diff
@@ -1,3 +1,17 @@
+terraform {
```

Comment:
This pins the versions for the main providers (e2e test is passing).
https://github.com/atlassian-labs/data-center-terraform/runs/5478975078?check_suite_focus=true#step:8:103

Reply:
We should put aws pin on modules/tfstate/main.tf

Reply:
Probably yes because that is a separate module.